wiki/Tournaments/RESC/2025/en.md

```diff
@@ -37,6 +37,8 @@ tags:
 | ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 25% of the prize pool, exclusive profile banner |
 | "4th place" | Exclusive profile banner |
 
+<!-- ![](img/badge-1st.png "Resurrection Cup 2025 winner badge") ![](img/badge-2nd.png "Resurrection Cup 2025 2nd-place badge") ![](img/badge-3rd.png "Resurrection Cup 2025 3rd-place badge") -->
+
 ## Organisation
 
 | Position | Member(s) |
```

Comment:
title case for the positions?

Reply:
https://github.com/ppy/osu-wiki/pull/13238#discussion_r2083277874